python/paddle/tensor/creation.py

```diff
@@ -1838,7 +1838,7 @@ def diagflat(x, offset=0, name=None):
 
     Examples:
         .. code-block:: python
-            :name: code-example-1
+            :name: code-example-a
```

Comment:
建议把 name 改成这种形式，可能以后就可以避免重复？
```suggestion
            :name: diagflat-example-1
```

Reply:
okk～